pkg/gce-pd-csi-driver/node.go

```diff
@@ -42,6 +44,18 @@ type GCENodeServer struct {
 
 var _ csi.NodeServer = &GCENodeServer{}
 
+// The constants are used to map from the machine type (number of CPUs) to the limit of
+// persistent disks that can be attached to an instance. Please refer to gcloud doc
+// https://cloud.google.com/compute/docs/disks/#increased_persistent_disk_limits
+const (
+	OneCPU               = 1
+	EightCPUs            = 8
+	VolumeLimit16 int64  = 16
+	VolumeLimit32 int64  = 32
+	VolumeLimit64 int64	 = 64
+	VolumeLimit128 int64 = 128
```

Comment:
Also from the updated page, it looks like the limits are now:

128 for 1 CPU and above
16 for shared-core

Reply:
Actually, the limits are determined more by memory now instead of CPU.

It seems like 3.75 GB is the cutoff for whether or not it's 16 or 128.